### PR TITLE
[plugin.video.rivedila7] v6.1.0

### PR DESCRIPTION
**Pull request by Testato**
### Description

- Improved Regex that repair many Programs view
- Added "Una giornata particolare" to Programs
- Filter Omnibus News vs Omnibus
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.rivedila7/.gitignore [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Byte-compiled / optimized / DLL files
```
Comment (basrieter) [inline plugin.video.rivedila7/.gitignore]: Exclude from your  submission please.

**Review by basrieter — CHANGES_REQUESTED** [review]
@luivit Are you OK with these changes? They seem to come from another dev? 
@Testato Please remove the `.gitignore`.

File: plugin.video.rivedila7/.gitignore (hunk 1 above)
Comment (Testato) [reply plugin.video.rivedila7/.gitignore]: Yes, I and Luivit are together the owner of this plugin. You can see it in the addon.xml

Why we need remove the .gitignore file ?

Comment (basrieter) [reply plugin.video.rivedila7/.gitignore]: Great. The .gitignore does not belong in a Kodi plugin. So it should not be included in the PR.

Comment (Testato) [reply plugin.video.rivedila7/.gitignore]: @basrieter 
Gitignore removed

File: plugin.video.rivedila7/resources/lib/plugin.py [hunk 2]
```diff
@@ -62,6 +62,9 @@ def add_directory_item_nodup(parameters, li, title='', folder=True, is_live=Fals
 
 
 def play_dirette(url, live):
+    xbmc.log('PLAY DIRETTE', xbmc.LOGINFO)
```
Comment (enen92) [inline plugin.video.rivedila7/resources/lib/plugin.py]: Please drop or change to debug

**Review by enen92 — CHANGES_REQUESTED** [review]
See comment

File: plugin.video.rivedila7/resources/lib/plugin.py (hunk 2 above)
Comment (Testato) [reply plugin.video.rivedila7/resources/lib/plugin.py]: Done

**Comment by Testato** [comment]
> See comment

loginfo dropped

**Comment by Testato** [comment]
> See comment

@enen92 @basrieter 
sorry but it's normal that a very simple update need 3 months for merging ?

**Review by enen92 — APPROVED** [review]

**Pull request merged**